### DISCUSSION TITLE
How about just JetBrains plugin?

### DISCUSSION THREAD

**Discussion by linux-china** [casey/just#1097] [Ideas]
This plugin was finished and waiting for JetBrains approval, and almost 2 business days.  Yes,  lex and bnf are hard to me,  and I try to refer code from other language plugins and read some books, please correct me if possible.  Source code repository:  https://github.com/linux-china/jetbrains-just-plugin  if you want to try early version, please visit https://github.com/linux-china/jetbrains-just-plugin/releases and download the plugin zip file and install manually. 

Some screenshot: 

![image](https://user-images.githubusercontent.com/46711/154779864-a81f3a00-36dc-4496-97d2-8af576023920.png)

![image](https://user-images.githubusercontent.com/46711/154779871-b98a1e2c-e9ba-406c-a048-3eeb251fce51.png)

![image](https://user-images.githubusercontent.com/46711/154779874-b6f6b450-cd75-45c6-b269-1084bae73875.png)

![image](https://user-images.githubusercontent.com/46711/154779882-2d1eb40e-5ef8-43f5-b904-a861f2e409b7.png)

Tested with https://github.com/casey/just/blob/master/justfile    and feedback are welcome.

**Comment by fzyzcjy** [comment]
Looking forward to the plugin!

**Reply by linux-china** [reply]
@fzyzcjy  it's available :)

**Reply by fzyzcjy** [reply]
Excellent job, thank you!

**Comment by linux-china** [comment]
JetBrains just plugin is available: https://plugins.jetbrains.com/plugin/18658-just
